### DISCUSSION TITLE
defaultNew with Office 2021 LTSC Standard

### DISCUSSION THREAD

**Discussion by Brosian** [Set-OutlookSignatures/Set-OutlookSignatures#77] [Q&A] [answered]
Hi, I am using Office 2021 LTSC Standard and have issues with setting the signature as default in Outlook. Signature is getting deployed and is visible in the list of signatures in Outlook but not set as default. Tried with **defaultNew** and **defaultReplyFwd**. Setup is hybrid AD with mail in 365.

Any ideas?

[log.txt](https://github.com/GruberMarkus/Set-OutlookSignatures/files/11556502/log.txt)

**Comment by GruberMarkus** [comment]
Hi @Brosian,

there may be several root causes:

Possible root cause 1:
```
Check trusts for open LDAP port and connectivity @2023-05-24T17:39:11+02:00@
  seq.domain.com
    LDAP query failed, remove domain from list.
    If this error is permanent, check firewalls, DNS and AD trust. Consider parameter TrustsToCheckForGroups.
```

Possible root cause 2:
```
Get AD properties of currently logged in user and assigned manager @2023-05-24T17:39:11+02:00@
  Currently logged in user
    User's mailbox not found in Outlook profiles, but Outlook Web signature and/or OOF message should be set. Add dummy mailbox entry.
    test.test@domain.com
[...]
Get AD properties of each mailbox @2023-05-24T17:39:12+02:00@
  Mailbox 'test.test@domain.com'
    Search for mailbox user object in domain/forest 'domain.com': Not found
  Mailbox 'test.test@domain.onmicrosoft.com'
    Search for mailbox user object in domain/forest 'domain.com': Not found
```

Root cause 1 and root cause 2 may be linked with each other, or be different issues.

I only had a quick look at the log, there may be more reasons.

You mentioned a hybrid connection between on-prem and Exchange Online. Is the Active Directory 'mail' attribute synced between Azure AD and on-prem AD?  
I can not clearly answer this question, as you have sanitized the log file. If you don't want to analyze it yourself, I need the unsanitized version (I can provide a secured upload location for you).

Is the current user located in the domain seq.domain.com?

The Outlook profile name '9375CFF0413111D3B88A00104B2A6676' is technically correct, but unusual. Is this really the correct name of the Outlook profile, and how come that this name has been chosen?

Which value does the following registry key have? `HKCU\Software\Microsoft\Office\16.0\Outlook\Profiles\9375CFF0413111D3B88A00104B2A6676\9375CFF0413111d3B88A00104B2A6676\00000001\New Signature`

**Reply by Brosian** [reply, marked as answer]
Hi Markus, thank you for your quick reply and looking into this.

I managed to get this to work now and it was our current signature solution that "locked" any changes made, so when I shutdown the process everything worked as intended.

I really appreciate your work and look forward using this!

Kind regards,
Peter

**Comment by GruberMarkus** [comment]
My pleasure!
